cmd/ocm/create/cluster/cmd.go

```diff
@@ -579,7 +579,7 @@ func promptName(argv []string) error {
 
 func promptClusterWideProxy() error {
 	var err error
-	if args.existingVPC.Enabled && !wasClusterWideProxyReceived() && args.interactive {
+	if !args.existingVPC.Enabled && !wasClusterWideProxyReceived() && args.interactive {
```

Comment:
Should this question be displayed regardless of if the existing vpc is enabled?

Reply:
no. Only if vpc is enabled

Reply:
so why is this check looking for `!args.existingVPC.Enabled`? that seems to mean the opposite, unless i'm missing something

Reply:
yup. you right. Thanks!